ceph.spec.in

```diff
@@ -782,14 +780,8 @@ Requires:	lvm2
 Requires:	sudo
 Requires:	libstoragemgmt
 Requires:	python%{python3_pkgversion}-ceph-common = %{_epoch_prefix}%{version}-%{release}
-%if 0%{?weak_deps}
-Recommends:	nvme-cli
-%if 0%{?suse_version}
-Requires:       smartmontools
-%else
-Recommends:	smartmontools
-%endif
-%endif
+Requires:	python%{python3_pkgversion}-setuptools
+
```

Comment:
Hi @cfsnyder, curious about this addition since `python%{python3_pkgversion}-setuptools` was not here originally, and it is not a part of the cherry-picked commit.

What exactly was the formatting conflict in ceph.spec.in?

Reply:
@cfsnyder ping

Reply:
hi @tchaikov I need a sanity check please :-) 
this `python%{python3_pkgversion}-setuptools` dependency should not be in ceph.spec.in file, right?

Reply:
Weird, I could have sworn I removed that  line. Sorry about that, updating it now @yaarith

Reply:
@cfsnyder thanks for the update; let's see the tests pass, then it's good to go!